### PR TITLE
Issue/reduce warning

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

- reduce pandas warning H -->h

## How Has This Been Tested?

CI tests

- [ ] Yes

## Checklist:

- [ ] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by peterdudfield** [comment]
Might want to remove pin, once #68 has been merged

File: pyproject.toml [hunk 1]
```diff
@@ -18,6 +18,7 @@ maintainers = [
 ]
 
 dependencies = [ # Migration from requirements.txt
+    "torch<2.5.0",
```
Comment (Sukh-P) [inline pyproject.toml:21]: Can remove this if this is merged after the remove of ocf_datapipes

Comment (peterdudfield) [reply pyproject.toml:21]: removed

Comment (AUdaltsova) [reply pyproject.toml:21]:
```suggestion
    "torch",
```
still shows up on changed files, so just to double-check?

Comment (peterdudfield) [reply pyproject.toml]: fixed now

**Pull request merged**